spec/features/admin/subscriptions_spec.rb

```diff
@@ -238,8 +238,7 @@ def open_subscription_products_panel
         expect(page).to have_input "ship_address_city", with: 'Natte Yallock'
         expect(page).to have_input "ship_address_zipcode", with: '3465'
         expect(page).to have_input "ship_address_phone", with: '0400 123 456'
-        expect(page).to have_input "ship_address_country_id", with: 'Australia'
-        expect(page).to have_input "ship_address_state_id", with: 'Victoria'
```

Comment:
I see this second commit removes these assertions. But I think it might be a good idea to check them as you did initially, only using a different matcher: `ship_address_country_id` and `ship_address_state_id` are selectors and input not fields, so I think `have_input` is triggering the error. One idea could be trying `have_selector` instead:
`expect(page).to have_selector '<dropdown_identifyer>', text: '<dropdown_option>'`